code/game/machinery/computer/Operating.dm

```diff
@@ -7,18 +7,19 @@
 	icon_keyboard = "med_key"
 	icon_screen = "crew"
 	circuit = /obj/item/circuitboard/operating
-	var/obj/machinery/optable/table = null
-	var/mob/living/carbon/human/victim = null
+	var/obj/machinery/optable/table
 	light_color = LIGHT_COLOR_PURE_BLUE
-	var/verbose = 1 //general speaker toggle
-	var/patientName = null
+	var/verbose = TRUE //general speaker toggle
 	var/oxyAlarm = 30 //oxy damage at which the computer will beep
-	var/choice = 0 //just for going into and out of the options menu
-	var/healthAnnounce = 1 //healther announcer toggle
-	var/crit = 1 //crit beeping toggle
+	var/choice = FALSE //just for going into and out of the options menu
+	var/healthAnnounce = TRUE //healther announcer toggle
+	var/crit = TRUE //crit beeping toggle
 	var/nextTick = OP_COMPUTER_COOLDOWN
 	var/healthAlarm = 50
-	var/oxy = 1 //oxygen beeping toggle
+	var/oxy = TRUE //oxygen beeping toggle
+	var/newPat = TRUE
+	var/patStat
+	var/patStatHolder
```

Comment:
The code style seems to be having full words for variables. So `var/patientStat` and `var/patientStatHolder`.
In either way, these need to be documented. on what they do - they are somewhat cryptic right now.

Reply:
patStat = Patient Status
patStatHolder = Patient Status Holder

Reply:
Leaving this comment open - not sure if current commits address this well enough.

`patientStatus` is intended to be a string that tells the computer what to say. The string is based on `table.patient.stat`. Is the patient Awake? Asleep? Dead?

`patientStatusHolder` holds the previous instance of `patientStatus`. When `patientStatus` no longer matches `patientStatusHolder`, it's time to update and have the Operating Computer inform the doctor.